sympy/matrices/decompositions.py

```diff
@@ -1122,6 +1121,222 @@ def _LUdecompositionFF(M):
 
     return P, L, DD, U
 
+def Singular_Value_Decomposition(A):
```

Comment:
```suggestion
def _singular_value_decomposition(A):
```
Why not rename like this like the functions in this file?